src/Orchard.Web/Modules/Orchard.Users/Extensions/MembershipSettingsExtensions.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Orchard.Security {
+    public static class MembershipSettingsExtensions {
+        public static int GetMinimumPasswordLength(this IMembershipSettings membershipSettings) {
+            return membershipSettings.EnableCustomPasswordPolicy && membershipSettings.EnableCustomPasswordLength ? membershipSettings.MinimumPasswordLength : 1;
```

Comment:
When applying this patch, will the default value become 1 instead of the current one (6 or 7).